src/embezzler.ts

```diff
@@ -761,8 +762,9 @@ export const embezzlerSources = [
         .map((source) => `${source.potential()} from ${source.name}`)
         .forEach((text) => print(text, HIGHLIGHT));
       globalOptions.askedAboutWish = true;
-      globalOptions.wishAnswer = userConfirm(
-        `Garbo has detected you have ${potential} potential ways to copy an Embezzler, but no way to start a fight with one. Current embezzler net (before potions) is ${averageEmbezzlerNet()}, so we expect to earn ${profit} meat, after the cost of a wish. Should we wish for an Embezzler?`
+      globalOptions.wishAnswer = customUserConfirm(
```

Comment:
I would argue that it is less destructive for these to default to `false`, but I can see why you'd want the default value to be otherwise